tools/rna_tools/intarna/test-data/intarna_result.csv

```diff
@@ -0,0 +1,5 @@
+id1;start1;end1;id2;start2;end2;subseqDP;hybridDP;E
```

Comment:
@zwanli since this is a pure output file, there is no problem with intarna.. the only problem might be due to wrong FASTA input headers including tab-whitespaces (that are not pruned by intarna) that might appear as sequence ids in the output and cause problems. but that's nothing we have to care about..

or you can do by putting two consecutive piping sed calls: the first converting tabs to space and the second converting semicolon to tab. that way the output should be always correct. :)

thanks,
Martin